### DISCUSSION TITLE
Nitpick: Cozy mode tab bar is hard to open on dual monitors

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#853] [Ideas]
### What happened?

This is super nitpicky, but the tab bar for Cozy mode is a little unforgiving with the mouse roller. It doesn't react very fast when my mouse rolls over it, and so when I accidentally overshoot it and have my mouse just barely on the second monitor, nothing happens and it takes me a second to realize nothing happened. The rollover zone is also pretty skinny, so it's easy to overshoot. I'm sure I'd get used to it, but it's a little frustrating.

My reference is Arc, which I've used for a while (But very much dislike that it uses chromium). It has a larger rollover zone, reacts quickly, and has a grace period when I overshoot so I can correct.

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0 a.26

### Severity impact

Low, Enhancement

### What platform are you seeing the problem on?

Windows

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/368_

**Comment by SlashScreen** [comment]
Not sure what else there is for me to add.
